lib/shelley/src/Cardano/Wallet/Shelley/Compatibility.hs

```diff
@@ -1004,6 +1010,35 @@ instance DecodeAddress 'Mainnet where
 instance DecodeAddress ('Testnet pm) where
     decodeAddress = _decodeAddress SL.Testnet
 
+decodeBytes :: Text -> Either TextDecodingError ByteString
```

Comment:
we already have `cardano-addresses` dependency in this project https://github.com/input-output-hk/cardano-wallet/blob/master/stack.yaml#L48 and also have many bits present here waiting for reuse. Probably not in thi sPR but I am in favor of strong reuse of cardano-addresses primitives..

Reply:
At some point yes, I agree. I started this PR with only cardano-addresses originally and I ended up reverting because errors were highly inconsistent between endpoints. So I prefer sticking to how it's done now (and also, locate the code doing it in the same place so it's either to change later).